### ISSUE TITLE
esprima-fb version 3001.0001.0000-dev-harmony-fb causes npm install errors

### ISSUE THREAD

**Issue by jimlloyd** [substack/astw#6]
I have a project that is no longer able to npm install. npm returns this error:

```
npm ERR! Darwin 14.1.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1

npm ERR! invalid version: 3001.0001.0000-dev-harmony-fb
```

The esprima-fb developer acknowledges the problem with this version but says it was fixed long ago, and suggests that I should contact the module owners that are still referencing this version. See issue: https://github.com/facebook/esprima/issues/105

The npm-debug.log file is 20K lines long, so is a PITA to analyze, but at the moment it seems that astw is the culprit. Can you please upgrade to a newer version?

In the meantime, I will try to pin down my dependency tree to see who is referring to your release with this problem.

**Comment by substack** [comment]
Published 1.3.0 that just depends on esprima `^2.1.0`.
For the 2.0.0 I'll just switch everything over to acorn.

**Comment by jimlloyd** [comment]
Thanks for the quick turnaround! It appears I now need you to update lexical-scope to refer to astw@1.3.0. I'll create an issue there.
